src/explore-education-statistics-admin/src/pages/publication/__tests__/PublicationContactPage.test.tsx

```diff
@@ -244,23 +260,15 @@ describe('PublicationContactPage', () => {
     userEvent.click(screen.getByRole('button', { name: 'Confirm' }));
 
     await waitFor(() => {
-      expect(publicationService.updatePublication).toHaveBeenCalledWith(
+      expect(publicationService.updateContact).toHaveBeenCalledWith(
         testPublication.id,
-        {
-          ...testPublication,
-          contact: {
-            contactName: 'John Smith',
-            contactTelNo: '0777777777',
-            teamEmail: 'john.smith@test.com',
-            teamName: 'Team Smith',
-          },
-        },
+        testContact,
```

Comment:
Can appreciate you didn't write this test originally, but it might be a good basic check to change one of the contact form's fields to check that the updated value gets passed through correctly.